app/javascript/mastodon/features/ui/components/navigation_panel.js

```diff
@@ -50,56 +50,57 @@ class NavigationPanel extends React.Component {
           <Link to='/' className='column-link column-link--logo'><Logo /></Link>
           <hr />
         </div>
-
-        {signedIn && (
-          <React.Fragment>
-            <ColumnLink transparent to='/home' icon='home' text={intl.formatMessage(messages.home)} />
-            <ColumnLink transparent to='/notifications' icon={<NotificationsCounterIcon className='column-link__icon' />} text={intl.formatMessage(messages.notifications)} />
-            <FollowRequestsColumnLink />
-          </React.Fragment>
-        )}
-
-        {showTrends ? (
-          <ColumnLink transparent to='/explore' icon='hashtag' text={intl.formatMessage(messages.explore)} />
-        ) : (
-          <ColumnLink transparent to='/search' icon='search' text={intl.formatMessage(messages.search)} />
-        )}
-
-        {(signedIn || timelinePreview) && (
-          <>
-            <ColumnLink transparent to='/public/local' icon='users' text={intl.formatMessage(messages.local)} />
-            <ColumnLink transparent exact to='/public' icon='globe' text={intl.formatMessage(messages.federated)} />
-          </>
-        )}
-
-        {!signedIn && (
-          <div className='navigation-panel__sign-in-banner'>
+        <div id='navigation-panel-links'>
```

Comment:
The change in this file was adding this `div` with the id for skip link compatibility. The other changes are from re-formatting that occurred as a result of this one new div.